fdbserver/include/fdbserver/TLogInterface.h

```diff
@@ -225,13 +221,16 @@ struct TLogPeekRequest {
 	bool onlySpilled;
 	Optional<std::pair<UID, int>> sequence;
 	ReplyPromise<TLogPeekReply> reply;
+	Optional<Version> end;
```

Comment:
maybe add a comment about this and specify if the version is inclusive or exclusive.